intTests/test_sawscript_builtins/files.log.good

```diff
@@ -25,5 +25,6 @@ do {
    let c = if true then {{ False }} else {{ True }};
    let (d, e) = (b, c);
    typedef t Int;
+   (return ());
```

Comment:
Just to make sure I understand the consequences of this change, does this mean that if you have a SAWScript file that contains just this:

```
typedef t Int;
```

Then that would be rejected, and and you would need to put a `return ()` statement afterwards to repair it? Or does that only apply to blocks that have an explicit `do` keyword?